src/frontend/src/pages/FinancePage/FinancePage.tsx

```diff
@@ -62,9 +62,13 @@ const FinancePage = () => {
   const [accountCreditModalShow, setAccountCreditModalShow] = useState<boolean>(false);
   const [showTotalAmountSpent, setShowTotalAmountSpent] = useState(false);
 
+  const [refundSource, setRefundSource] = useState('');
   const [startDate, setStartDate] = useState(new Date());
   const [endDate, setEndDate] = useState(new Date());
 
+  const [endDatePickerOpen, setEndDatePickerOpen] = useState(false);
+  const [startDatePickerOpen, setStartDatePickerOpen] = useState(false);
```

Comment:
these shouldn't have been moved from the other file, they're only used there so they should stay there